### PR TITLE
reactive form

### PR DESCRIPTION
**Pull request by MadhuraDevi28**

### CODE REVIEW

File: src/app/student-login/student-login.component.html [hunk 1]
```diff
@@ -1,16 +1,16 @@
+<form [ FormGroup ]="loginForm"(ng submit="login()")>
```
Comment (kishore2904) [inline src/app/student-login/student-login.component.html]: (ngSubmit) = "login()"

Check the format

Comment (MadhuraDevi28) [reply src/app/student-login/student-login.component.html]: changed

File: src/app/student-login/student-login.component.html [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 <div class="body">
     <div class="login-container">
         <h2>studentsLogin</h2>
-     <form>
-         <input type="text" placeholder="Username or Email" name="username_email" required><br>
-         <input type="password" placeholder="Password" name="password" required><br>
-         <button type="submit">Login</button>
+        <input type="text" placeholder="Username or Email" name="username_email" required, formcontrolName="username"><br>
+         <input type="password" placeholder="Password" name="password" required, formcontrolName="password"><br>
```
Comment (kishore2904) [inline src/app/student-login/student-login.component.html]: why required needed here we can add it in the formcoontrol

Comment (kishore2904) [inline src/app/student-login/student-login.component.html]: formControlName not formcontrolName

**Pull request closed without merging**